lasagne/layers/conv.py

```diff
@@ -744,7 +747,11 @@ def __init__(self, incoming, num_filters, filter_size, stride=(1, 1),
                  crop=0, untie_biases=False,
                  W=init.GlorotUniform(), b=init.Constant(0.),
                  nonlinearity=nonlinearities.rectify, flip_filters=False,
-                 **kwargs):
+                 output_size=None, **kwargs):
+        if (not isinstance(output_size, T.TensorVariable) and
+                output_size is not None):
+            output_size = as_tuple(output_size, 2, int)
+        self.output_size = output_size
```

Comment:
Maybe add a comment: `# output_size must be set before calling the super constructor`